react/tsconfig.json

```diff
@@ -0,0 +1,37 @@
+{
+    "compilerOptions": {
+        "baseUrl": "src",
+        "paths": {
+            "test-utils": [
+                "./tests/js"
+            ]
+        },
+        "outDir": "./dist", // path to output directory
+        "sourceMap": true, // allow sourcemap support
+        "strict": true, // enable strict null checks as a best practice
+        // These (and more) are turned on by strict
+        //"strictNullChecks": true, // enable strict null checks as a best practice
+        //"noImplicitAny": true,
+        //"noImplicitThis": true,
+        //"strictPropertyInitialization": true,
+        // Should probable have these 'linting' options enabled
```

Comment:
```suggestion
        "strict": true, // enable all of the strict mode family of options
```

Reply:
Slight fix for misplaced comment and removal of unneeded (and commented out) options